index.html

```diff
@@ -15,6 +15,23 @@
 
   <body>
     <div class="container">
+
+        <header>
+            <nav>
+                <a href="index.html">
+                    <img class="logo" src="img/lambda-black.png" alt="lambda rocks!!"> </a>
```

Comment:
Great use of wrapping the Lambda logo and making it a responsive link that redirects back to the home page.